client/src/views/mapping/components/inputColumn/addJoinButton.tsx

```diff
@@ -0,0 +1,36 @@
+import { Button } from "@blueprintjs/core";
+
+import * as React from "react";
```

Comment:
i prefer importing only the symbols needed from react

Reply:
After some checks, this import is mandatory.
If missing, the TSX compiler make this error : `TS2686: 'React' refers to a UMD global, but the current file is a module. Consider adding an import instead.`

Reply:
I think if you merge from master, this error should now go away.